src/corelibs/U2View/src/ov_msa/find_pattern/FindPatternMsaWidgetFactory.cpp

```diff
@@ -32,7 +34,7 @@ namespace U2 {
 
 const QString FindPatternMsaWidgetFactory::GROUP_ID = "OP_MSA_FIND_PATTERN";
 const QString FindPatternMsaWidgetFactory::GROUP_ICON_STR = ":core/images/find_dialog.png";
-const QString FindPatternMsaWidgetFactory::GROUP_DOC_PAGE = "666";
+const QString FindPatternMsaWidgetFactory::GROUP_DOC_PAGE = HelpButton::INVALID_VALUE;
```

Comment:
Do not initialize static variables with another static variables if they are in different compilation units. It will cause an undefined behavior.